### PR TITLE
[CHI-2019] Add Chicago 2019 dates

### PR DESCRIPTION
**Pull request by mattstratton**
Pending countersigned contract from venue.

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
This references `/events/2018-chicago/` for a number of assets. Probably want that to be `2019` instead.

File: data/events/2019-chicago.yml [hunk 1]
```diff
@@ -6,13 +6,13 @@ description: "Devopsdays is coming to Chicago!" # Edit this to suit your prefere
 ga_tracking_id: "UA-74738648-1" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
 
 # All dates are in unquoted 2019-MM-DD or YYYY-MM-DDTHH:MM:SS+TZ:TZ, like this:   variable: 2016-01-05T23:59:59+02:00
-startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
-enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+startdate: 2019-08-27 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2019-08-28 # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
 cfp_date_start:  # start accepting talk proposals.
```
Comment (bridgetkromhout) [inline data/events/2019-chicago.yml]: The build errors out if it can't find `cfp_date_start`.

Comment (bridgetkromhout) [reply data/events/2019-chicago.yml]: 11:18:47 AM: ERROR 2019/02/09 17:18:47 Unable to locate template for shortcode 'param' in page propose
11:18:49 AM: ERROR 2019/02/09 17:18:49 theme/partials/footer.html template: theme/partials/footer.html:50:30: executing "theme/partials/footer.html" at <time .cfp_date_start>: error calling time: Unable to Cast <nil> to Time
11:18:59 AM: ERROR 2019/02/09 17:18:59 theme/partials/speaking.html template: theme/partials/speaking.html:4:24: executing "theme/partials/speaking.html" at <time .cfp_date_start>: error calling time: Unable to Cast <nil> to Time

Comment (mattstratton) [reply data/events/2019-chicago.yml]: Good catch. I think I fixed it.

**Comment by bridgetkromhout** [comment]
Just noticed, @mattstratton - you seem to have some unrelated commits in here.

**Comment by mattstratton** [comment]
@bridgetkromhout it's possible they got pulled in on a rebase somehow; I was trying to get your commits into my local version. There's so much in here right now but I am a little worried about trying to pull out those extras.

**Comment by bridgetkromhout** [comment]
Ah, I recommend a `git merge master` on this PR's branch, to bring it up to date, and then push the branch up. I tried locally (didn't push anything) and there were no conflicts.

**Comment by mattstratton** [comment]
Closing this PR while we finalize venue contract

**Pull request closed without merging**